action/has_vm.go

```diff
@@ -20,7 +18,7 @@ func NewHasVM(
 func (a HasVMAction) Run(vmCID VMCID) (bool, error) {
 	_, found, err := a.vmFinder.Find(int(vmCID))
 	if err != nil {
-		return false, bosherr.WrapErrorf(err, "Finding VM '%d'", vmCID)
+		return false, nil
```

Comment:
CPI should allow VMNotFoundError otherwise it will block the deployment. 
The action is compliance with the specification of CPI API v1
http://bosh.io/docs/cpi-api-v1.html